src/js/angular/core/services/license.service.js

```diff
@@ -23,6 +23,8 @@ function licenseService($rootScope, LicenseRestService, $translate) {
                 that.loadingLicense = false;
                 updateProductType(that.license);
             });
+        }).finally(function () {
+            $rootScope.$broadcast('license.set');
```

Comment:
Create a file /core/global-events.js and put an exported constant LICENSE_SET there. Then reuse it. See this for an example https://github.com/Ontotext-AD/graphdb-workbench/blob/master/src/js/angular/clustermanagement/events.js.

EDIT: 
Above should be done only if actually this broadcast is proper here, which I think is not the case. 
Given the code in the method before this line, it's obsious that the license might actually not be set when the finally clause is hit.
Another issue with this code is that the service is bound to the $rootScope which is bad, because services should be unaware of the UI. In other words, if this broadcast should be executed, then is should be done in the license controller most likely.